Source/WebKit/UIProcess/XR/ios/PlatformXRARKit.mm

```diff
@@ -125,14 +125,14 @@
                 .presentationSession = WTFMove(presentationSession),
                 .renderThread = Thread::create("ARKitCoordinator session renderer", [this] { renderLoop(); }),
                 .renderSemaphore = Box<BinarySemaphore>::create(),
+                .terminateRequested = false,
             };
```

Comment:
So this could be like 

```
struct RenderParameters {
        BinarySemaphore semaphore;
        std::atomic<bool> shouldQuit;
        ... sessionEventClient;
        ... presentationSession;
        ....;   
  };
  Box<RenderParameters> params = new RenderParameters {
            ....= WTFMove(sessionEventClient),
            ....= WTFMove(presentationSession),
  };


   m_state = Active {
                .pageIdentifier = page.webPageID(),
                .params = params;  
                .renderThread = Thread::create("ARKitCoordinator session renderer", [this, params] { renderLoop(*params); }),
             };
```